pkg/services/auth.go

```diff
@@ -10,6 +10,22 @@ import (
 	"github.com/cmsgov/mint-app/pkg/graph/model"
 )
 
+// HasAnyRole authorizes a user as having any in a collection of given roles
+func HasAnyRole(ctx context.Context, roles []model.Role) (bool, error) {
+	for _, role := range roles {
+		userHasRole, err := HasRole(ctx, role)
+		if err != nil {
+			return userHasRole, err
```

Comment:
```suggestion
			return false, err
```

Wondering if this might be safer - if `HasRole()` returns an error, it might be safe to assume that we shouldn't grant any roles (though, practically, it doesn't look like `HasRole()` ever actually returns an error... maybe we should just nix the `error` return from both of these functions and simplify them?

Reply:
Even if there is no current error implementation I think it's best to preserve the error state so that any systems using this method will already handle future error implementations.

I am on the fence about returning values on error states, I seem to recall we do both in different areas of the code base. In this case I agree that returning false is likely more rational. It would be good to have a discussion and project-scope decision on how we handle returning captured values vs default values on errors.